### DISCUSSION TITLE
Peaks overlapping multiple genes and becoming mislabeled after sample integration.

### DISCUSSION THREAD

**Discussion by coopershawna** [stuart-lab/signac#750] [General]
The peak for one of my main marker genes has become broader (Macs3 called originally) after integration of 2 scATAC samples. It now overlaps part of another gene and is being mislabeled as that other gene.  What can I do to fix this?

**Comment by coopershawna** [comment]
I am using the original "peaks" assay for my differential accessibility and the coordinates match between peaks in my lists and what I see in the coverage plot.  
Thank you for your wisdom and I really appreciate the ability to ask questions!

**Comment by timoast** [comment]
Hi @coopershawna

When merging different ChromatinAssay objects, we do the following:

1. Check if the peaks are identical across the objects. If they are identical, we do not alter the peak genomic ranges.
2. If peaks are not identical, we need to find "equivalent" regions across the assays by finding peak regions across the two assays that overlap. Otherwise we can have a situation where slightly different peak calls causes the same region to be treated as completely different features in downstream analysis.
3. Merge overlapping peaks and adjust peak coordinates accordingly

Wherever possible, I recommend quantifying exactly the same set of peaks across any assays that you want to merge. This is the most accurate approach, and will prevent peaks being adjusted after merging the objects. The adjustment of peaks that we do when merging assays that do not contain the same peak sets is not ideal, but is the best we can do in that situation (it's better than treating overlapping peaks as completely distinct features).
